Clinical-Genomics/cg
# Upload delivery report via scout

This PR fixes broken way we get genes from genepanels from scout

How to test:
0. install on stage (add instructions on how to as needed)
1. create delivery-report

Expected outcome:
delivery-report created
